build.sbt

```diff
@@ -70,8 +53,8 @@ lazy val kamonAkka25 = Project("kamon-akka-25", file("kamon-akka-2.5.x"))
   .settings(Seq(
     bintrayPackage := "kamon-akka",
     moduleName := "kamon-akka-2.5",
-    scalaVersion := "2.12.1",
-    crossScalaVersions := Seq("2.11.8", "2.12.1"),
+    scalaVersion := "2.12.7",
+    crossScalaVersions := Seq("2.11.8", "2.12.7"),
     resolvers += Resolver.bintrayRepo("kamon-io", "snapshots")))
   .enablePlugins(JavaAgent)
   .settings(javaAgents ++= resolveAgent)
```

Comment:
This is not needed since the latest sbt umbrella. You only need to leave the `.enablePlugins(JavaAgent)` and that's it.